src/main/java/frc/robot/RobotManager.java

```diff
@@ -69,7 +72,9 @@ public void teleopInit() {
 
 	@Override
 	public void robotPeriodic() {
-		TimeUtils.updateCycleTime(); // Better to be first
+		roborioCycles++; // Better to be first
+		Logger.recordOutput("RoborioCycles", roborioCycles);
+		TimeUtils.updateCycleTime(); // Better to be second
```

Comment:
the comment states that this should be second, but it is not the second line of the function. Do one of the following:
1. move to be the second line
2. change comment to `//Better to be third`
3. clarify the meaning of `second` in the comment(something like `// Better to be second non logging statement` [this specific comment is rather confusing, so you should probably try to think of something clearer])

Reply:
i dont think its neccessery to do any of these
i mean its kinda like common sense to log something after you change it and i dont think we should add any more comments
in my opinion its just fine like this

Reply:
maybe move logging of `roborioCycles` to reside with all other loggings (after `CommandScheduler.getInstance().run();`

Reply:
i think its better to log right after change in the value